tasks/task453_swag_answer_generation.json

```diff
@@ -6,7 +6,10 @@
         "SWAG: (dataset: https://github.com/rowanz/swagaf/tree/master/data, paper: https://arxiv.org/pdf/1808.05326.pdf)"
     ],
     "Categories": [
-        "Answer Generation"
+        "Text Generation -> Sentence Generation -> Story Completion"
```

Comment:
This task is not exactly sentence generation. So maybe just "Story Completion" or "Text Generation -> Story Completion" are better options.